src/driver/rtc.h

```diff
@@ -0,0 +1,72 @@
+#ifndef _RTC_H
+#define _RTC_H
```

Comment:
we have a recent enough gcc, lets please go with a `#pragma once`

Reply:
Excellent!  I will make the change.